### ISSUE TITLE
Infinite recursion in getDistance for both CANUltrasonicDistanceSensor and CANInfraredDistanceSensor

### ISSUE THREAD

**Issue by leijurv** [RoboticsTeam4904/standard#119]
```
@Override
	public double getDistance() {
		try {
			return getDistance();
		}
		catch (Exception e) {
			LogKitten.ex(e);
			return 0;
		}
	}
```
org.usfirst.frc4904.standard.custom.sensors.CANInfraredDistanceSensor.getDistance
org.usfirst.frc4904.standard.custom.sensors.CANUltrasonicDistanceSensor.getDistance

**Comment by leijurv** [comment]
@c0d3rman @carturn @carterturn

**Comment by carturn** [comment]
Fixed in #128.
